etl/steps/data/garden/countries/2023-09-29/cow_ssm.meta.yml

```diff
@@ -23,31 +38,9 @@ tables:
         description_short: |-
           Number of states listed in the state system membership list for a given region.
```

Comment:
also possibly a more informative description_short here - e.g. `The number of states in a region's state system membership list depends on changing sovereign state criteria: before 1919, a population above 500,000 and recognition by France and the UK were required; after 1919, either League of Nations or UN membership, or a population over 500,000 recognized by two major powers, was necessary.`

Reply:
this is already captured in `definitions.common.description_key`